.github/container/test-t5x.sh

```diff
@@ -31,6 +32,7 @@ fi
 
 ADDITIONAL_ARGS=""
 BATCH_SIZE=0
+USE_CONTRIB_CONFIGS=0
```

Comment:
Ha.. you are already using this name internally.

Reply:
Yea, I think I realized that half way in and missed it in the usage. Thanks for catching